axelrod/tests/unit/test_titfortat.py

```diff
@@ -462,5 +462,31 @@ def test_effect_of_strategy(self):
         self.responses_test([C]*3, [C, D, C], [C])
         self.responses_test([C]*3, [C, D, D], [D])
 
+class TestAdaptiveTitForTat(TestPlayer):
 
+    name = "Adaptive Tit For Tat: 0.5"
+    player = axelrod.AdaptiveTitForTat
+    expected_classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+    
+    def test_strategy(self):
+        """Start by cooperating."""
+        self.first_play_test(C)
+        
+    def test_effect_of_strategy(self):
+        
+        self.markov_test(['C', 'D', 'C', 'D'])
+        
+        p1, p2 = self.player(), self.player()
+        p1.play(p2)
+        p1.play(p2)
+        self.assertEqual(p2.world, 0.75)
+        
```

Comment:
Can we have an extra test for the reset? Just to check that the value of the rate and world go back to the defaults?

If you call it `test_reset` it will overwrite the test here: https://github.com/Axelrod-Python/Axelrod/blob/master/axelrod/tests/unit/test_player.py#L163

So perhaps call it `test_world_and_rate_reset` (or something like that) :)